float64[] goal1
float64[] goal2
float64 velocity
float64 acceleration

---

string status